ambari/src/main/java/io/brooklyn/ambari/rest/AmbariConfig.java

```diff
@@ -0,0 +1,36 @@
+package io.brooklyn.ambari.rest;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.collect.ImmutableList;
+
+public class AmbariConfig {
+
+    private boolean hasHosts = false;
+    private Map<String, List<String>> hostGroups = new HashMap<String, List<String>>();
+    private Map<String, List<String>> hostGroupsToHosts = new HashMap<String, List<String>>();
+
+    public void add(
+            String groupName,
+            List<String> hostFQDNs,
+            List<String> components) {
+        hostGroups.put(groupName, ImmutableList.copyOf(components));
+        hostGroupsToHosts.put(groupName, ImmutableList.copyOf(hostFQDNs));
```

Comment:
I'll add some checks but my current thinking is that the config object is temporary and will be removed when we clean up the calls to ambari.